src/components/AdminPage.jsx

```diff
@@ -0,0 +1,32 @@
+import { useContext } from 'react';
+import { useNavigate } from 'react-router-dom';
+import AuthContext from '../Store/auth-context';
+
+function AdminPage() {
+    const navigate = useNavigate();
+    const authCtx = useContext(AuthContext);
+
+    const logoutHandler = () => {
+        authCtx.logout();
+        navigate('/', { replace: true });
+    }
+
+    return (
+        <div className="adminheader-container">
+            <div className="adminheader">
+                <h4>OVERZICHT</h4>
+                <h2>Reserveringen</h2>
+            </div>
+
+            <nav className='navbar'>
+                {/* <button className='request-button' > to user homepage</button>
+                <button className='request-button' > boek toevoegen </button>
+                <button className='request-button' > werknemer toevoegen</button>
+                <span className="my-spacer">Voornaam Achternaam</span> */}
```

Comment:
Hier staat nog een comment. Moeten we hier iets mee?

Reply:
Yess, als ik css ooit minder vervloek. Dit zijn alle knoppen die straks gaan verwijzen naar de juiste pagina's. Maar ze wilden maar niet op de juiste plek komen die ik wilde. Ga ik vandaag mee verder